cmd/clusterlint/main.go

```diff
@@ -105,6 +105,11 @@ func main() {
 			},
 			Action: runChecks,
 		},
+		{
+			Name:   "groups",
```

Comment:
I wonder if we should make this a subcommand of `clusterlint list` - i.e., have `clusterlint list checks` and `clusterlint list groups`.